.esh/commands/locales-full.py

```
@@ -29,200 +29,200 @@
+	full = yaml.safe_load(in_path.read_text(encoding="utf-8"))
+	main_rel = f"src/public/locales/{lang}.json"
+	wrote = 0
+
+	if "_docs_readme" in full:
+		readme_path = DOCS_DIR / f"Readme.{lang}.md"
+		if write_if_changed(readme_path, _readme_from_yaml(full["_docs_readme"])):
+			print(f"Wrote readme: {readme_path.name}")
+			wrote += 1
+
+	if main_rel in full:
+		if write_if_changed(FOUNT_DIR / main_rel, _json_str(full[main_rel])):
```

Ruff (0.16.0)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)